sdk/nodejs/examples/http-server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": {
+    "commander": "^11.1.0",
+    "express": "^4.18.2",
+    "path": "^0.12.7",
+    "readline-sync": "^1.4.10",
+    "@openziti/zrok": "^0.1.0"
```

Comment:
We aren't establishing a new version scheme are we? `0.1.0`? Can we use the main zrok repository tag version for this?